src/Components/DynamicFields.res

```diff
@@ -435,11 +436,22 @@ let make = (
               options=currencyArr
             />
           | FullName =>
-            <FullNamePaymentInput
-              paymentType
-              customFieldName={item->getCustomFieldName}
-              optionalRequiredFields={Some(requiredFields)}
-            />
+            <>
+              <div
```

Comment:
Could you please move this inside FullNamePaymentInput

Reply:
Nope it can't be moved as this will affect everywhere where we have used this that's why I haven't done.